src/Text/Parsing/Parser/Token.purs

```diff
@@ -75,50 +75,49 @@ type LanguageDef = GenLanguageDef String Identity
 -- | The `GenLanguageDef` type is a record that contains all parameterizable
 -- | features of the "Text.Parsec.Token" module. The module `Text.Parsec.Language`
 -- | contains some default definitions.
-newtype GenLanguageDef s m
-  = LanguageDef
+newtype GenLanguageDef s m = LanguageDef
   {
     -- | Describes the start of a block comment. Use the empty string if the
     -- | language doesn't support block comments. For example `/*`.
     commentStart :: String
-  ,
-    -- | Describes the end of a block comment. Use the empty string if the
-    -- | language doesn't support block comments. For example `*/`.
-    commentEnd :: String
-  ,
-    -- | Describes the start of a line comment. Use the empty string if the
-    -- | language doesn't support line comments. For example `//`.
-    commentLine :: String
-  ,
-    -- | Set to `true` if the language supports nested block comments.
-    nestedComments :: Boolean
-  ,
-    -- | This parser should accept any start characters of identifiers. For
-    -- | example `letter <|> char '_'`.
-    identStart :: ParserT s m Char
-  ,
-    -- | This parser should accept any legal tail characters of identifiers.
-    -- | For example `alphaNum <|> char '_'`.
-    identLetter :: ParserT s m Char
-  ,
-    -- | This parser should accept any start characters of operators. For
-    -- | example `oneOf [':', '+', '=']`.
-    opStart :: ParserT s m Char
-  ,
-    -- | This parser should accept any legal tail characters of operators.
-    -- | Note that this parser should even be defined if the language doesn't
-    -- | support user-defined operators, or otherwise the `reservedOp`
-    -- | parser won't work correctly.
-    opLetter :: ParserT s m Char
-  ,
-    -- | The list of reserved identifiers.
-    reservedNames :: Array String
-  ,
-    -- | The list of reserved operators.
-    reservedOpNames :: Array String
-  ,
-    -- | Set to `true` if the language is case sensitive.
-    caseSensitive :: Boolean
+
+  -- | Describes the end of a block comment. Use the empty string if the
+  -- | language doesn't support block comments. For example `*/`.
+  , commentEnd :: String
```

Comment:
@natefaubion It'd be nice if the comments could align with the identifier, not the comma -- thoughts?

Reply:
Comments align to the token they are attached to.

Reply:
Due to the comma, will these constructors have documentation once docs are pushed to Pursuit?